docs/usage.md

```diff
@@ -220,42 +227,42 @@ For example:
 > :warning: This params is deprecated -- it will be removed in a future release.
 > Please check: [`--input`](#--input)
 
-### --sampleDir
```

Comment:
Still some part that are deleted, sorry about all that

Reply:
No it's my fault, I'll try again